keyboards/squarebuilt/readme.md

```diff
@@ -0,0 +1,15 @@
+# squarebuilt
+
+![squarebuilt](https://i0.wp.com/lancemercado.com/wp-content/uploads/2019/06/img_20190504_165635.jpg?resize=525%2C700&ssl=1)
+
+A short description of the keyboard/project
+
+Keyboard Maintainer: [LanceMercado](https://github.com/LanceMercado)  
+Hardware Supported: The PCBs, controllers supported  
+Hardware Availability: links to where you can find this hardware
```

Comment:
This line should be filled out with a link to a purchase vector (like a web storefront) or a GitHub repo where someone can download the files to have the PCB manufactured on their own.